addons/binding/org.openhab.binding.pentair/src/main/java/org/openhab/binding/pentair/handler/PentairBaseBridgeHandler.java

```diff
@@ -0,0 +1,470 @@
+                                break;
+                            }
+
+                            buf[1] = 0x2;
+                            length = 3;
+                            // assume 3 byte command, plus 1 checksum, plus 0x10, 0x03
+                            if (getBytes(buf, 2, 6) != 6) {
+                                break;
+                            }
+
+                            // Check to see if this is a 3 or 4 byte command
+                            if ((buf[6] != (byte) 0x10 || buf[7] != (byte) 0x03)) {
+                                length = 4;
+
+                                buf[8] = (byte) getByte(parserstate);
+                                if ((buf[7] != (byte) 0x10) && (buf[8] != (byte) 0x03)) {
+                                    logger.debug("Invalid Intellichlor command: {}",
+                                            PentairPacket.bytesToHex(buf, length + 6));
+                                    break; // invalid command
+                                }
+                            }
+
+                            chksum = 0;
+                            for (i = 0; i < length + 2; i++) {
+                                chksum += buf[i] & 0xFF;
+                            }
+
+                            c = buf[length + 2] & 0xFF;
+                            if (c != (chksum & 0xFF)) { // make sure it matches chksum
+                                logger.debug("Invalid Intellichlore checksum: {}",
+                                        PentairPacket.bytesToHex(buf, length + 6));
+                                break;
+                            }
+
+                            PentairPacket_Intellichlor pic = new PentairPacket_Intellichlor(buf, length);
+
+                            thing = findThing(0);
+
+                            if (thing == null) {
+                                if (!m_unregistered.contains(0)) { // if not yet seen, print out log message
+                                    logger.info("Command from unregistered Intelliflow: {}", pic.toString());
+                                    m_unregistered.add(0);
+                                } else {
+                                    logger.trace("Command from unregistered Intelliflow: {}", pic.toString());
+                                }
+
+                                break;
+                            }
+
+                            thinghandler = (PentairBaseThingHandler) thing.getHandler();
+                            if (thinghandler == null) {
+                                logger.debug("Thing handler = null");
+                                break;
+                            }
+
+                            thinghandler.processPacketFrom(pic);
+
+                            break;
+                    }
+                }
+            } catch (IOException e) {
+                logger.error("I/O error while reading from stream: {}", e.getMessage());
```

Comment:
remove this logging, but add the message as a third parameter to the updateStatus call